### PR TITLE
[5.3] Changed changeLocale event string to a class 

### PR DESCRIPTION
**Pull request by morloderex**
Sense 5.2 all other events have been converted to classes but this one still remains to be consistent with all the other events in the system i have changed this to also be a class.

**Please note that i haven't updated the tests for the application class sense i was unsure of how to do that!**

### CODE REVIEW

File: src/Illuminate/Translation/Events/LocaleChanged.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+     * Create a new event instance.
+     *
+     * @param $locale
+     * @return void
+     */
+    public function __construct($locale)
+    {
+        $this->locale = $locale;
+    }
+}
```
Comment (Macsch15) [inline src/Illuminate/Translation/Events/LocaleChanged.php]: Missing new line at the EOF :+1:

Comment (morloderex) [inline src/Illuminate/Translation/Events/LocaleChanged.php]: Added Thanks! 👍

File: src/Illuminate/Translation/Events/LocaleChanged.php [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+     * The changed locale.
+     *
+     * @var $locale
+     */
+    public $locale;
+
+    /**
+     * Create a new event instance.
+     *
+     * @param $locale
```
Comment (Macsch15) [inline src/Illuminate/Translation/Events/LocaleChanged.php]: @param in docblock should have also datatype (in this case - string)

**Comment by taylorotwell** [comment]
Holding off on this.

**Comment by taylorotwell** [comment]
Tests are failing.

**Comment by GrahamCampbell** [comment]
Looks like a mock just needs updating.

**Comment by morloderex** [comment]
As noted above i am not really sure how to update the mock.

**Comment by taylorotwell** [comment]
I'll hold off on this for next release. I already have a lot of things to document and we're very close to release.

**Comment by morloderex** [comment]
@taylorotwell should i open a new pr for master branch instead then?

**Comment by GrahamCampbell** [comment]
I'd vote 👍 for still PRing to master. It's upto Taylor ultimately though. :)

**Comment by morloderex** [comment]
I'll try to fix the mocking and return with a useful pr next time :)

**Pull request closed without merging**